README.md

```diff
@@ -1,22 +1,67 @@
-# dokku-pr-delete-db-mongo Dokku Plugin
+# Dokku Plugins
 
-This repository contains a script designed to efficiently manage the deletion of preview applications. These applications follow a defined naming convention. As part of the deletion process, the script ensures that the corresponding database, which is named identically to the app, is properly handled. The database, typically powered by MongoDB, is required for the app's functionality. The script verifies that the database is running on a specific server, which is identified by a URL sourced from a predefined configuration. This streamlined process ensures that both the preview app and its associated database are effectively and consistently managed, facilitating the organization's development workflows.
+Various [Dokku](https://dokku.com) plugins used to develop and manage software at [Code for Africa](https://codeforafrica.org).
+
+## Available Plugins
+
+### `pr-db-mongo`
 
 ## Prerequisites
 
 - [Dokku](https://dokku.com/docs/development/plugin-triggers)
 - [Python](https://www.python.org/downloads/)
+- Mongo Database Tools.
+  > To install mongo database tools,
+
+```sh
+    wget -qO - https://www.mongodb.org/static/pgp/server-4.4.asc | sudo apt-key add -
+    echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/4.4 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-4.4.list
+    sudo apt-get update
+    sudo apt-get install -y mongodb-org
+```
 
 ## Installation
 
+The .tgz installation via dokku plugin:install file://<path-to-tgz> or https://<path-to-tgz> seem to fail. We shall therefore use the manual installation until a fix is found.
+
+### Copy the plugin(s) to dev
+
+Copy the .tgz from the local/CI instance to the dev machine (running Dokku)
+
 ```bash
-dokku plugin:install https://github.com/CodeForAfrica/dokku-pr-delete-db-mongo
+scp plugin.tgz user@remotehost:/path-to-copy-file
 ```
 
-## Usage
+For example, if our plugin is called `pr-db-mongo`, then:
 
-### Environment Variables
+```bash
+scp build/pr-db-mongo.tgz ubuntu@dev.codeforafrica.org:/~
+```
 
-## Contributing
+### Make the plugin available to Dokku
 
-To contribute, clone the repository and add your scripts
+ssh to the dev machine
+
+```bash
+ssh ubuntu@dev.codeforafrica.org:
+```
+
+Extract and copy plugin into `$PLUGIN_AVAILABLE_PATH`
+
+```
+tar -xf pr-db-mongo.tgz
+chown dokku:dokku -R pr-db-mongo
+mv pr-db-mongo /var/lib/dokku/plugins/available
+```
+
+### Enable the plugin
+
+```bash
+sudo dokku plugin:enable plugin-name
+```
+
+That's it. Happy coding!
```

Comment:
Don't we need to run `plugin:install` and/or `plugin:install-dependencies`?

Reply:
We can only run `plugin:install-dependencies`

Reply:
We can run `plugin:install`, the question is do we need to? There is that `install` script that must run before any other trigger.

Reply:
Then we need it